### ISSUE TITLE
New enchantment for attack speed

### ISSUE THREAD

**Issue by Buxixx** [Tslat/Advent-Of-Ascension#2522]
**Describe the new feature you think would be good to add**

If there are enchantments for swords, greatblades to increase attack damage, there can be enchantment to increase the other parameter, the attack speed. Name suggestion: Rapid I, Rapid II, etc. This can be apply to any ranged weapon too.

**Add any alternative implementations you think might work**
Is there a slightly different way you think might work as well?

**Include any additional background information**
Does this suggestion relate to anything existing? Has it been talked about previously?

**Comment by DiamondWalker** [comment]
Good idea, although it doesn't feel like it fits AoA, and it might cause balance issues.

Maybe the enchantment could be exclusive for greatblades or guns, since those would probably benefit more from them anyways (swords already have decent attack speed).

**Comment by Golden-Extreme** [comment]
Is it possible to enchant with Maul?
